packages/caver-account/src/accountKey/accountKeyPublic.d.ts

```diff
@@ -0,0 +1,86 @@
+/**
+ * Representing an AccountKeyPublic.
+ * @class
+ */
+export interface IAccountKeyPublic {
```

Comment:
Why should interface and class be defined in d.ts file at the same time? What purpose is each used for?

Reply:
@jimni1222 
yes. I made it.

To make instance of class in as a class property, I had to define interface
You can refer to [Typescript Docs](https://www.typescriptlang.org/docs/handbook/interfaces.html#class-types)
For example, if you set type as class(not interface), it returns error
```
export class Caver {
    ...
    account: Account // not account: IAccount
    ...
}
```
`
This expression is not constructable.
  Type 'Account' has no construct signatures.
`